incubator/hnc/pkg/kubectl/configset.go

```diff
@@ -0,0 +1,65 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+var setTypeCmd = &cobra.Command{
+	Use: fmt.Sprintf("set-type <%s|%s|%s>", string(api.Propagate), string(api.Remove),
```

Comment:
Is the `string(x)` cast really needed?

Also, the usage should be something more like `set-type --kind X --apiVersion Y <propagate|remove|ignore>`, right?

Reply:
It doesn't look like we need the `string(x)` cast. Removed them. Changed to the usage to contain the flags.